arbnode/dataposter/externalsignertest/externalsignertest.go

```diff
@@ -38,6 +39,12 @@ type CertAbsPaths struct {
 	ClientKey  string
 }
 
+type SignerServer struct {
+	*http.Server
+	*SignerAPI
+	l net.Listener
```

Comment:
please call that listener.. or something else with more than 1 letter :)

Reply:
Done